.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+
+python:
+  - "3.4"
+
+services:
+  - postgresql
+
+addons:
+  postgresql: "9.6"
+
+before_install:
+  - python --version
+  - pip --version
+  - pg_config --version
+  - psql --version
+
+  - wget -q https://github.com/electrum/tpch-dbgen/archive/32f1c1b92d1664dba542e927d23d86ffa57aa253.zip -O tpch-dbgen.zip
+  - unzip -q tpch-dbgen.zip
+  - mv tpch-dbgen-32f1c1b92d1664dba542e927d23d86ffa57aa253 tpch-dbgen
+  - rm tpch-dbgen.zip
+
+install:
+  - pip install -r requirements.txt
+  - pip install psycopg2-binary
```

Comment:
Is there a reason why we don't put this in requirements?

Reply:
The idea was that psycopg2-binary is just a fix for a deprecation warning during build. It has nothing to do with the functionality in the code.

Reply:
From the [release documentation](http://initd.org/psycopg/articles/2018/02/08/psycopg-274-released/)

> Starting with Psycopg 2.7.4, we are releasing two different packages on PyPI: psycopg2 and psycopg2-binary. The latter is a Wheels-only package, with a behaviour identical to the classic one – the different name is used only in installation (it is installed by pip install psycopg2-binary, but still imported with import psycopg2 in Python).  
For the lifespan of the Psycopg 2.7 cycle, the psycopg2 PyPI package will still contain wheel packages, but starting from Psycopg 2.8 it will become again a source-only package. Starting from Psycopg 2.7.4, if the package is installed as binary from the psycopg2 PyPI entry, a warning will be reported on import.

Judging by this I think we are safe to put only the `psycopg2-binary` package in the requirements.